### PR TITLE
don't fail remove_metadata if fragment annotation doesn't exist

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-7246

Signed-off-by: Robert Cerven <rcerven@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by athos-ribeiro** [comment]
There is a typo on the commit message (s/metada/metadata)

File: atomic_reactor/plugins/exit_remove_worker_metadata.py [hunk 1]
```diff
@@ -31,7 +31,11 @@ def run(self):
 
         for platform, build_annotations in worker_builds.items():
             try:
-                cm_key, _ = get_platform_config(platform, build_annotations)
+                if ('metadata_fragment' in build_annotations and
+                        'metadata_fragment_key' in build_annotations):
+                    cm_key, _ = get_platform_config(platform, build_annotations)
+                else:
+                    continue
```
Comment (twaugh) [inline atomic_reactor/plugins/exit_remove_worker_metadata.py]: Wouldn't need an `else` keyword if the condition was reversed:
```
if ('metadata_fragment' not in build_annotations or
        'metadata_fragment_key' not in build_annotations):
    continue

cm_key, _ = ...
```

Comment (rcerven) [reply atomic_reactor/plugins/exit_remove_worker_metadata.py]: changed

**Comment by rcerven** [comment]
not updating rn, fix for unreleased feature

**Pull request merged**